polaris.shopify.com/content/components/choice-list.md

```diff
@@ -108,86 +109,41 @@ Shipping Options
 
 <!-- end -->
 
-- Not use colons
+### Not use colons
 
 <!-- usageblock -->
 
 #### Do
 
 If the customer abandons their checkout, send them an email reminder to complete their order
 
-<p>
-  <label><input type="radio" name="foo"> Never</label><br />
-  <label><input type="radio" name="foo"> 6 hours later</label><br />
-  <label><input type="radio" name="foo"> 24 hours later</label>
-</p>
-
 #### Don’t
 
 If the customer abandons their checkout, send them an email reminder to complete their order:
 
-<p>
-  <label><input type="radio" name="bar"> Never</label><br />
-  <label><input type="radio" name="bar"> 6 hours later</label><br />
-  <label><input type="radio" name="bar"> 24 hours later</label>
-</p>
 <!-- end -->
 
 ### List choices
 
 Every item in a choice list should:
 
 - Start with a capital letter
-
-<!-- usageblock -->
-
-#### Do
-
-- Option 1
-- Option 2
-- Option 3
-
-#### Don’t
-
-- option 1
-- option 2
-- option 3
```

Comment:
Did you remove these because they are too obvious?

Reply:
Going to add them back. But yeah there is some not very helpful examples in this component.

Reply:
The documentation was super duplicative here. I have consolidated the three do/don't examples into one.

Reply:
No super strong opinions, I think this might be the wrong cure. I think it's clearer to only focus on one thing at a time.

Other thoughts:

- Maybe this section doesn't need dos and don'ts?
- Down the road, I think many dos and don't could be visually smaller

Anyways, ship what you see fit!